k8s-resources.tf

```diff
@@ -55,5 +61,9 @@ resource "kubernetes_service_account" "alb_ingress_controller" {
       "eks.amazonaws.com/role-arn" = "arn:aws:iam::${var.account_id}:role/${aws_iam_role.alb-ingress-controller-iam-role.name}"
     }
   }
-  automount_service_account_token = true
+  secret {
+    name = kubernetes_secret.alb_ingress_controller.name
```

Comment:
Putting the names of manually created secrets in the `secrets` list in the ServiceAccount object is not correct, and will result in warnings about use of auto-generated secrets.

This list in the API contains the names of secrets to be _mounted_ into pods, not the names of manually created token secrets for use outside of pods.